### ISSUE TITLE
Ubuntu 20.04 subiquity log

### ISSUE THREAD

**Issue by BuhtigithuB** [boxcutter/ubuntu#207]
Hi,

I saw the other opened issue... But none report full extensive log output... Here my debug output : 

```
 packer build -only=virtualbox-iso -var-file=ubuntu2004-desktop.json ubuntu.json
2022/04/06 18:23:28 [INFO] Packer version: 1.8.0 [go1.17.8 windows amd64]
2022/04/06 18:23:28 [TRACE] discovering plugins in C:...\hashicorp\packer
2022/04/06 18:23:28 [TRACE] discovering plugins in C:...\AppData\Roaming\packer.d\plugins
2022/04/06 18:23:28 [TRACE] discovering plugins in .
2022/04/06 18:23:28 [INFO] PACKER_CONFIG env var not set; checking the default config file path
2022/04/06 18:23:28 [INFO] PACKER_CONFIG env var set; attempting to open config file: C:...\AppData\Roaming\packer.config
2022/04/06 18:23:28 [WARN] Config file doesn't exist: C:...\AppData\Roaming\packer.config
2022/04/06 18:23:28 [INFO] Setting cache directory: C:...\boxcutter\ubuntu\packer_cache
e: cannot determine if process is in background: Process background check error: not implemented yet
2022/04/06 18:23:28 [TRACE] Starting internal plugin packer-builder-virtualbox-iso
2022/04/06 18:23:28 Starting plugin: C:...\hashicorp\packer\packer.exe []string{"C:\...\\hashicorp\\packer\\packer.exe", "plugin", "packer-builder-virtualbox-iso"}
2022/04/06 18:23:28 Waiting for RPC address for: C:...\hashicorp\packer\packer.exe
2022/04/06 18:23:29 packer.exe plugin: [INFO] Packer version: 1.8.0 [go1.17.8 windows amd64]
2022/04/06 18:23:29 packer.exe plugin: [INFO] PACKER_CONFIG env var not set; checking the default config file path
2022/04/06 18:23:29 packer.exe plugin: [INFO] PACKER_CONFIG env var set; attempting to open config file: C:...\AppData\Roaming\packer.config
2022/04/06 18:23:29 packer.exe plugin: [WARN] Config file doesn't exist: C:...\AppData\Roaming\packer.config
2022/04/06 18:23:29 packer.exe plugin: [INFO] Setting cache directory: C:...\boxcutter\ubuntu\packer_cache
2022/04/06 18:23:29 packer.exe plugin: args: []string{"packer-builder-virtualbox-iso"}
2022/04/06 18:23:29 packer.exe plugin: Plugin port range: [10000,25000]
2022/04/06 18:23:29 packer.exe plugin: Plugin address: tcp 127.0.0.1:10000
2022/04/06 18:23:29 packer.exe plugin: Waiting for connection...
2022/04/06 18:23:29 Received tcp RPC address for C:...\hashicorp\packer\packer.exe: addr is 127.0.0.1:10000
2022/04/06 18:23:29 packer.exe plugin: Serving a plugin connection...
2022/04/06 18:23:29 [TRACE] Starting internal plugin packer-provisioner-shell
2022/04/06 18:23:29 Starting plugin: C:...\hashicorp\packer\packer.exe []string{"C:\...\\hashicorp\\packer\\packer.exe", "plugin", "packer-provisioner-shell"}
2022/04/06 18:23:29 Waiting for RPC address for: C:...\hashicorp\packer\packer.exe
2022/04/06 18:23:29 packer.exe plugin: [INFO] Packer version: 1.8.0 [go1.17.8 windows amd64]
2022/04/06 18:23:29 packer.exe plugin: [INFO] PACKER_CONFIG env var not set; checking the default config file path
2022/04/06 18:23:29 packer.exe plugin: [INFO] PACKER_CONFIG env var set; attempting to open config file: C:...\AppData\Roaming\packer.config
2022/04/06 18:23:29 packer.exe plugin: [WARN] Config file doesn't exist: C:...\AppData\Roaming\packer.config
2022/04/06 18:23:29 packer.exe plugin: [INFO] Setting cache directory: C:...\boxcutter\ubuntu\packer_cache
2022/04/06 18:23:29 packer.exe plugin: args: []string{"packer-provisioner-shell"}
2022/04/06 18:23:29 packer.exe plugin: Plugin port range: [10000,25000]
2022/04/06 18:23:29 packer.exe plugin: Plugin address: tcp 127.0.0.1:10000
2022/04/06 18:23:29 Received tcp RPC address for C:...\hashicorp\packer\packer.exe: addr is 127.0.0.1:10000
2022/04/06 18:23:29 packer.exe plugin: Waiting for connection...
2022/04/06 18:23:29 packer.exe plugin: Serving a plugin connection...
2022/04/06 18:23:29 [TRACE] Starting internal plugin packer-post-processor-vagrant
2022/04/06 18:23:29 Starting plugin: C:...\hashicorp\packer\packer.exe []string{"C:\...\\hashicorp\\packer\\packer.exe", "plugin", "packer-post-processor-vagrant"}
2022/04/06 18:23:29 Waiting for RPC address for: C:...\hashicorp\packer\packer.exe
2022/04/06 18:23:29 packer.exe plugin: [INFO] Packer version: 1.8.0 [go1.17.8 windows amd64]
2022/04/06 18:23:29 packer.exe plugin: [INFO] PACKER_CONFIG env var not set; checking the default config file path
2022/04/06 18:23:29 packer.exe plugin: [INFO] PACKER_CONFIG env var set; attempting to open config file: C:...\AppData\Roaming\packer.config
2022/04/06 18:23:29 packer.exe plugin: [WARN] Config file doesn't exist: C:...\AppData\Roaming\packer.config
2022/04/06 18:23:29 packer.exe plugin: [INFO] Setting cache directory: C:...\boxcutter\ubuntu\packer_cache
2022/04/06 18:23:29 packer.exe plugin: args: []string{"packer-post-processor-vagrant"}
2022/04/06 18:23:29 packer.exe plugin: Plugin port range: [10000,25000]
2022/04/06 18:23:29 packer.exe plugin: Plugin address: tcp 127.0.0.1:10000
2022/04/06 18:23:29 packer.exe plugin: Waiting for connection...
2022/04/06 18:23:29 Received tcp RPC address for C:...\hashicorp\packer\packer.exe: addr is 127.0.0.1:10000
2022/04/06 18:23:29 packer.exe plugin: Serving a plugin connection...
2022/04/06 18:23:29 Preparing build: virtualbox-iso
2022/04/06 18:23:29 Build debug mode: false
2022/04/06 18:23:29 Force build: false
2022/04/06 18:23:29 On error:
2022/04/06 18:23:29 Waiting on builds to complete...
2022/04/06 18:23:29 Starting build run: virtualbox-iso
virtualbox-iso: output will be in this color.
2022/04/06 18:23:29 Running builder: virtualbox-iso

2022/04/06 18:23:29 [INFO] (telemetry) Starting builder virtualbox-iso
2022/04/06 18:23:29 packer.exe plugin: [DEBUG] builder/virtualbox: VBOX_MSI_INSTALL_PATH = C:\Program Files\Oracle\VirtualBox\
2022/04/06 18:23:29 packer.exe plugin: VBoxManage path: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe
2022/04/06 18:23:29 packer.exe plugin: VBoxManage --version output: 6.1.32r149290
2022/04/06 18:23:29 packer.exe plugin: VirtualBox version: 6.1.32
2022/04/06 18:23:29 packer.exe plugin: guest_additions_url is blank; querying driver for iso.
2022/04/06 18:23:29 packer.exe plugin: Found Default Guest Additions ISO: C:\Program Files\Oracle\VirtualBox/VBoxGuestAdditions.iso
2022/04/06 18:23:29 packer.exe plugin: Guest additions URL: C:\Program Files\Oracle\VirtualBox/VBoxGuestAdditions.iso
==> virtualbox-iso: Retrieving Guest additions
==> virtualbox-iso: Trying C:\Program Files\Oracle\VirtualBox/VBoxGuestAdditions.iso
2022/04/06 18:23:29 packer.exe plugin: Acquiring lock for: file://C:/Program%20Files/Oracle/VirtualBox/VBoxGuestAdditions.iso (C:...\boxcutter\ubuntu\packer_cache\afa7695602909b267b26b23c8aefe12db6ee78b2.iso.lock)
==> virtualbox-iso: Trying file://C:/Program%20Files/Oracle/VirtualBox/VBoxGuestAdditions.iso
2022/04/06 18:23:29 packer.exe plugin: Leaving retrieve loop for Guest additions
==> virtualbox-iso: file://C:/Program%20Files/Oracle/VirtualBox/VBoxGuestAdditions.iso => C:/Program Files/Oracle/VirtualBox/VBoxGuestAdditions.iso
==> virtualbox-iso: Retrieving ISO
==> virtualbox-iso: Trying /Volumes/Storage/software/ubuntu/ubuntu-20.04.4-live-server-amd64.iso
2022/04/06 18:23:29 packer.exe plugin: Acquiring lock for: /Volumes/Storage/software/ubuntu/ubuntu-20.04.4-live-server-amd64.iso?checksum=sha256%3A28ccdb56450e643bad03bb7bcf7507ce3d8d90e8bf09e38f6bd9ac298a98eaad (C:...\boxcutter\ubuntu\packer_cache\47de2d7266acde194681de2a24f5d76b43b452ca.iso.lock)
==> virtualbox-iso: Trying /Volumes/Storage/software/ubuntu/ubuntu-20.04.4-live-server-amd64.iso?checksum=sha256%3A28ccdb56450e643bad03bb7bcf7507ce3d8d90e8bf09e38f6bd9ac298a98eaad
2022/04/06 18:23:33 packer.exe plugin: Leaving retrieve loop for ISO
==> virtualbox-iso: /Volumes/Storage/software/ubuntu/ubuntu-20.04.4-live-server-amd64.iso?checksum=sha256%3A28ccdb56450e643bad03bb7bcf7507ce3d8d90e8bf09e38f6bd9ac298a98eaad => C:...\boxcutter\ubuntu\packer_cache\47de2d7266acde194681de2a24f5d76b43b452ca.iso
2022/04/06 18:23:33 packer.exe plugin: No floppy files specified. Floppy disk will not be made.
2022/04/06 18:23:33 packer.exe plugin: No CD files specified. CD disk will not be made.
2022/04/06 18:23:33 packer.exe plugin: Found available port: 8874 on IP: 0.0.0.0
==> virtualbox-iso: Starting HTTP server on port 8874
2022/04/06 18:23:33 packer.exe plugin: Suppressing annoying messages in VirtualBox
2022/04/06 18:23:33 packer.exe plugin: Executing VBoxManage: []string{"setextradata", "global", "GUI/RegistrationData", "triesLeft=0"}
2022/04/06 18:23:33 packer.exe plugin: stdout:
2022/04/06 18:23:33 packer.exe plugin: stderr:
2022/04/06 18:23:33 packer.exe plugin: Executing VBoxManage: []string{"setextradata", "global", "GUI/SuppressMessages", "confirmInputCapture,remindAboutAutoCapture,remindAboutMouseIntegrationOff,remindAboutMouseIntegrationOn,remindAboutWrongColorDepth"}
2022/04/06 18:23:33 packer.exe plugin: stdout:
2022/04/06 18:23:33 packer.exe plugin: stderr:
2022/04/06 18:23:33 packer.exe plugin: Executing VBoxManage: []string{"setextradata", "global", "GUI/UpdateCheckCount", "60"}
2022/04/06 18:23:33 packer.exe plugin: stdout:
2022/04/06 18:23:33 packer.exe plugin: stderr:
2022/04/06 18:23:33 packer.exe plugin: Executing VBoxManage: []string{"setextradata", "global", "GUI/UpdateDate", "1 d, 2023-01-01, stable"}
2022/04/06 18:23:33 packer.exe plugin: stdout:
2022/04/06 18:23:33 packer.exe plugin: stderr:
==> virtualbox-iso: Creating virtual machine...
2022/04/06 18:23:33 packer.exe plugin: Executing VBoxManage: []string{"createvm", "--name", "ubuntu2004-desktop", "--ostype", "Ubuntu_64", "--register"}
2022/04/06 18:23:33 packer.exe plugin: stdout: Virtual machine 'ubuntu2004-desktop' is created and registered.
2022/04/06 18:23:33 packer.exe plugin: UUID: 9f848954-8318-4d5f-8204-ad8a73d3f72e
2022/04/06 18:23:33 packer.exe plugin: Settings file: 'C:...\VirtualBox VMs\ubuntu2004-desktop\ubuntu2004-desktop.vbox'
2022/04/06 18:23:33 packer.exe plugin: stderr:
2022/04/06 18:23:33 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--boot1", "disk", "--boot2", "dvd", "--boot3", "none", "--boot4", "none"}
2022/04/06 18:23:34 packer.exe plugin: stdout:
2022/04/06 18:23:34 packer.exe plugin: stderr:
2022/04/06 18:23:34 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--cpus", "1"}
2022/04/06 18:23:34 packer.exe plugin: stdout:
2022/04/06 18:23:34 packer.exe plugin: stderr:
2022/04/06 18:23:34 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--memory", "512"}
2022/04/06 18:23:34 packer.exe plugin: stdout:
2022/04/06 18:23:34 packer.exe plugin: stderr:
2022/04/06 18:23:34 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--usb", "off"}
2022/04/06 18:23:34 packer.exe plugin: stdout:
2022/04/06 18:23:34 packer.exe plugin: stderr:
2022/04/06 18:23:34 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--audio", "none", "--audiocontroller", "ac97"}
2022/04/06 18:23:34 packer.exe plugin: stdout:
2022/04/06 18:23:34 packer.exe plugin: stderr:
2022/04/06 18:23:34 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--chipset", "piix3"}
2022/04/06 18:23:34 packer.exe plugin: stdout:
2022/04/06 18:23:34 packer.exe plugin: stderr:
2022/04/06 18:23:34 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--firmware", "bios"}
2022/04/06 18:23:34 packer.exe plugin: stdout:
2022/04/06 18:23:34 packer.exe plugin: stderr:
2022/04/06 18:23:34 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--nictype1", "82540EM", "--nictype2", "82540EM", "--nictype3", "82540EM", "--nictype4", "82540EM", "--nictype5", "82540EM", "--nictype6", "82540EM", "--nictype7", "82540EM", "--nictype8", "82540EM"}
2022/04/06 18:23:34 packer.exe plugin: stdout:
2022/04/06 18:23:34 packer.exe plugin: stderr:
2022/04/06 18:23:34 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--graphicscontroller", "vboxvga", "--vram", "4"}
2022/04/06 18:23:34 packer.exe plugin: stdout:
2022/04/06 18:23:34 packer.exe plugin: stderr:
2022/04/06 18:23:34 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--rtcuseutc", "off"}
2022/04/06 18:23:34 packer.exe plugin: stdout:
2022/04/06 18:23:34 packer.exe plugin: stderr:
2022/04/06 18:23:34 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--accelerate3d", "off"}
2022/04/06 18:23:34 packer.exe plugin: stdout:
2022/04/06 18:23:34 packer.exe plugin: stderr:
2022/04/06 18:23:34 packer.exe plugin: Executing VBoxManage: []string{"createhd", "--filename", "output-ubuntu2004-desktop-virtualbox-iso\\ubuntu2004-desktop.vdi", "--size", "130048", "--format", "VDI", "--variant", "Standard"}
==> virtualbox-iso: Creating hard drive output-ubuntu2004-desktop-virtualbox-iso\ubuntu2004-desktop.vdi with size 130048 MiB...
2022/04/06 18:23:35 packer.exe plugin: stdout: Medium created. UUID: 5e9d8201-5d34-4608-8141-7762e74bff35
2022/04/06 18:23:35 packer.exe plugin: stderr: 0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
2022/04/06 18:23:35 packer.exe plugin: Executing VBoxManage: []string{"storagectl", "ubuntu2004-desktop", "--name", "IDE Controller", "--add", "ide"}
2022/04/06 18:23:35 packer.exe plugin: stdout:
2022/04/06 18:23:35 packer.exe plugin: stderr:
2022/04/06 18:23:35 packer.exe plugin: VBoxManage --version output: 6.1.32r149290
2022/04/06 18:23:35 packer.exe plugin: VirtualBox version: 6.1.32
2022/04/06 18:23:35 packer.exe plugin: Executing VBoxManage: []string{"storagectl", "ubuntu2004-desktop", "--name", "SATA Controller", "--add", "sata", "--portcount", "1"}
2022/04/06 18:23:35 packer.exe plugin: stdout:
2022/04/06 18:23:35 packer.exe plugin: stderr:
2022/04/06 18:23:35 packer.exe plugin: Executing VBoxManage: []string{"storageattach", "ubuntu2004-desktop", "--storagectl", "SATA Controller", "--port", "0", "--device", "0", "--type", "hdd", "--medium", "output-ubuntu2004-desktop-virtualbox-iso\\ubuntu2004-desktop.vdi", "--nonrotational", "off", "--discard", "off"}
2022/04/06 18:23:35 packer.exe plugin: stdout:
2022/04/06 18:23:35 packer.exe plugin: stderr:
==> virtualbox-iso: Mounting ISOs...
2022/04/06 18:23:35 packer.exe plugin: Not attaching guest additions since we're uploading.
    virtualbox-iso: Mounting boot ISO...
2022/04/06 18:23:35 packer.exe plugin: Executing VBoxManage: []string{"storageattach", "ubuntu2004-desktop", "--storagectl", "IDE Controller", "--port", "0", "--device", "1", "--type", "dvddrive", "--medium", "C:\...\\boxcutter\\ubuntu\\packer_cache\\47de2d7266acde194681de2a24f5d76b43b452ca.iso"}
2022/04/06 18:23:35 packer.exe plugin: stdout:
2022/04/06 18:23:35 packer.exe plugin: stderr:
2022/04/06 18:23:35 packer.exe plugin: Looking for available port between 5900 and 6000 on 127.0.0.1
2022/04/06 18:23:35 packer.exe plugin: Found available port: 5992 on IP: 127.0.0.1
2022/04/06 18:23:35 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--vrdeaddress", "127.0.0.1", "--vrdeauthtype", "null", "--vrde", "on", "--vrdeport", "5992"}
2022/04/06 18:23:35 packer.exe plugin: stdout:
2022/04/06 18:23:35 packer.exe plugin: stderr:
2022/04/06 18:23:35 packer.exe plugin: No floppy disk, not attaching.
2022/04/06 18:23:35 packer.exe plugin: Looking for available communicator (SSH, WinRM, etc) port between 2222 and 4444
2022/04/06 18:23:35 packer.exe plugin: Found available port: 4358 on IP: 127.0.0.1
2022/04/06 18:23:35 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--nic1", "nat"}
2022/04/06 18:23:35 packer.exe plugin: stdout:
2022/04/06 18:23:35 packer.exe plugin: stderr:
==> virtualbox-iso: Creating forwarded port mapping for communicator (SSH, WinRM, etc) (host port 4358)
2022/04/06 18:23:35 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--natpf1", "packercomm,tcp,127.0.0.1,4358,,22"}
2022/04/06 18:23:35 packer.exe plugin: stdout:
2022/04/06 18:23:35 packer.exe plugin: stderr:
==> virtualbox-iso: Executing custom VBoxManage commands...
    virtualbox-iso: Executing: modifyvm ubuntu2004-desktop --nictype1 virtio
2022/04/06 18:23:35 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--nictype1", "virtio"}
2022/04/06 18:23:35 packer.exe plugin: stdout:
2022/04/06 18:23:35 packer.exe plugin: stderr:
    virtualbox-iso: Executing: modifyvm ubuntu2004-desktop --memory 8000
2022/04/06 18:23:35 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--memory", "8000"}
2022/04/06 18:23:36 packer.exe plugin: stdout:
2022/04/06 18:23:36 packer.exe plugin: stderr:
    virtualbox-iso: Executing: modifyvm ubuntu2004-desktop --cpus 4
2022/04/06 18:23:36 packer.exe plugin: Executing VBoxManage: []string{"modifyvm", "ubuntu2004-desktop", "--cpus", "4"}
2022/04/06 18:23:36 packer.exe plugin: stdout:
2022/04/06 18:23:36 packer.exe plugin: stderr:
==> virtualbox-iso: Starting the virtual machine...
2022/04/06 18:23:36 packer.exe plugin: Executing VBoxManage: []string{"startvm", "ubuntu2004-desktop", "--type", "gui"}
2022/04/06 18:23:45 packer.exe plugin: stdout: Waiting for VM "ubuntu2004-desktop" to power on...
2022/04/06 18:23:45 packer.exe plugin: VM "ubuntu2004-desktop" has been successfully started.
2022/04/06 18:23:45 packer.exe plugin: stderr:
==> virtualbox-iso: Waiting 5s for boot...
==> virtualbox-iso: Typing the boot command...
2022/04/06 18:23:50 packer.exe plugin: Special code 'Press' '<enter>' found, replacing with: &{[1c] [9c]}
2022/04/06 18:23:50 packer.exe plugin: Special code 'Press' '<enter>' found, replacing with: &{[1c] [9c]}
2022/04/06 18:23:50 packer.exe plugin: Special code 'Press' '<f6>' found, replacing with: &{[40] [c0]}
2022/04/06 18:23:50 packer.exe plugin: Special code 'Press' '<esc>' found, replacing with: &{[01] [81]}
2022/04/06 18:23:50 packer.exe plugin: Executing VBoxManage: []string{"controlvm", "ubuntu2004-desktop", "keyboardputscancode", "1c", "9c", "1c", "9c", "40", "c0", "01", "81"}
2022/04/06 18:23:51 packer.exe plugin: stdout:
2022/04/06 18:23:51 packer.exe plugin: stderr:
2022/04/06 18:23:51 packer.exe plugin: [INFO] Waiting 1s
2022/04/06 18:23:52 packer.exe plugin: Sending char 'a', code '1e9e', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'u', code '1696', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 't', code '1494', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'o', code '1898', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'i', code '1797', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'n', code '31b1', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 's', code '1f9f', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 't', code '1494', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'a', code '1e9e', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'l', code '26a6', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'l', code '26a6', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char ' ', code '39b9', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'd', code '20a0', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 's', code '1f9f', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '=', code '0d8d', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'n', code '31b1', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'o', code '1898', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'c', code '2eae', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'l', code '26a6', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'o', code '1898', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'u', code '1696', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'd', code '20a0', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '-', code '0c8c', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'n', code '31b1', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'e', code '1292', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 't', code '1494', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char ';', code '27a7', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 's', code '1f9f', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '=', code '0d8d', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'h', code '23a3', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 't', code '1494', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 't', code '1494', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char 'p', code '1999', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char ':', code '2a27a7aa', shift true
2022/04/06 18:23:52 packer.exe plugin: Sending char '/', code '35b5', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '/', code '35b5', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '1', code '0282', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '0', code '0b8b', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '.', code '34b4', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '0', code '0b8b', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '.', code '34b4', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '2', code '0383', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '.', code '34b4', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '2', code '0383', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char ':', code '2a27a7aa', shift true
2022/04/06 18:23:52 packer.exe plugin: Sending char '8', code '0989', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '8', code '0989', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '7', code '0888', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '4', code '0585', shift false
2022/04/06 18:23:52 packer.exe plugin: Sending char '/', code '35b5', shift false
2022/04/06 18:23:52 packer.exe plugin: Executing VBoxManage: []string{"controlvm", "ubuntu2004-desktop", "keyboardputscancode", "1e", "9e", "16", "96", "14", "94", "18", "98", "17", "97", "31", "b1", "1f", "9f", "14", "94", "1e", "9e", "26", "a6", "26", "a6", "39", "b9"}
2022/04/06 18:23:52 packer.exe plugin: stdout:
2022/04/06 18:23:52 packer.exe plugin: stderr:
2022/04/06 18:23:52 packer.exe plugin: Executing VBoxManage: []string{"controlvm", "ubuntu2004-desktop", "keyboardputscancode", "20", "a0", "1f", "9f", "0d", "8d", "31", "b1", "18", "98", "2e", "ae", "26", "a6", "18", "98", "16", "96", "20", "a0", "0c", "8c", "31", "b1"}
2022/04/06 18:23:52 packer.exe plugin: stdout:
2022/04/06 18:23:52 packer.exe plugin: stderr:
2022/04/06 18:23:52 packer.exe plugin: Executing VBoxManage: []string{"controlvm", "ubuntu2004-desktop", "keyboardputscancode", "12", "92", "14", "94", "27", "a7", "1f", "9f", "0d", "8d", "23", "a3", "14", "94", "14", "94", "19", "99", "2a", "27", "a7", "aa", "35", "b5"}
2022/04/06 18:23:53 packer.exe plugin: stdout:
2022/04/06 18:23:53 packer.exe plugin: stderr:
2022/04/06 18:23:53 packer.exe plugin: Executing VBoxManage: []string{"controlvm", "ubuntu2004-desktop", "keyboardputscancode", "35", "b5", "02", "82", "0b", "8b", "34", "b4", "0b", "8b", "34", "b4", "03", "83", "34", "b4", "03", "83", "2a", "27", "a7", "aa", "09", "89"}
2022/04/06 18:23:53 packer.exe plugin: stdout:
2022/04/06 18:23:53 packer.exe plugin: stderr:
2022/04/06 18:23:53 packer.exe plugin: Executing VBoxManage: []string{"controlvm", "ubuntu2004-desktop", "keyboardputscancode", "09", "89", "08", "88", "05", "85", "35", "b5"}
2022/04/06 18:23:53 packer.exe plugin: stdout:
2022/04/06 18:23:53 packer.exe plugin: stderr:
2022/04/06 18:23:54 packer.exe plugin: [INFO] Waiting 1s
2022/04/06 18:23:55 packer.exe plugin: Special code 'Press' '<enter>' found, replacing with: &{[1c] [9c]}
2022/04/06 18:23:55 packer.exe plugin: Executing VBoxManage: []string{"controlvm", "ubuntu2004-desktop", "keyboardputscancode", "1c", "9c"}
2022/04/06 18:23:55 packer.exe plugin: stdout:
2022/04/06 18:23:55 packer.exe plugin: stderr:
==> virtualbox-iso: Using SSH communicator to connect: 127.0.0.1
2022/04/06 18:23:55 packer.exe plugin: [INFO] Waiting for SSH, up to timeout: 2h46m40s
==> virtualbox-iso: Waiting for SSH to become available...
2022/04/06 18:23:55 packer.exe plugin: [INFO] Attempting SSH connection to 127.0.0.1:4358...
2022/04/06 18:23:55 packer.exe plugin: [DEBUG] reconnecting to TCP connection for SSH
2022/04/06 18:23:55 packer.exe plugin: [DEBUG] handshaking with SSH
2022/04/06 18:24:55 packer.exe plugin: [DEBUG] SSH handshake err: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none password], no supported methods remain
2022/04/06 18:24:55 packer.exe plugin: [DEBUG] Detected authentication error. Increasing handshake attempts.
2022/04/06 18:25:02 packer.exe plugin: [INFO] Attempting SSH connection to 127.0.0.1:4358...
2022/04/06 18:25:02 packer.exe plugin: [DEBUG] reconnecting to TCP connection for SSH
2022/04/06 18:25:02 packer.exe plugin: [DEBUG] handshaking with SSH
2022/04/06 18:25:04 packer.exe plugin: [DEBUG] SSH handshake err: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none password], no supported methods remain
2022/04/06 18:25:04 packer.exe plugin: [DEBUG] Detected authentication error. Increasing handshake attempts.
2022/04/06 18:25:11 packer.exe plugin: [INFO] Attempting SSH connection to 127.0.0.1:4358...
2022/04/06 18:25:11 packer.exe plugin: [DEBUG] reconnecting to TCP connection for SSH
2022/04/06 18:25:11 packer.exe plugin: [DEBUG] handshaking with SSH
2022/04/06 18:25:15 packer.exe plugin: [DEBUG] SSH handshake err: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none password], no supported methods remain
2022/04/06 18:25:15 packer.exe plugin: [DEBUG] Detected authentication error. Increasing handshake attempts.
2022/04/06 18:25:22 packer.exe plugin: [INFO] Attempting SSH connection to 127.0.0.1:4358...
2022/04/06 18:25:22 packer.exe plugin: [DEBUG] reconnecting to TCP connection for SSH
2022/04/06 18:25:22 packer.exe plugin: [DEBUG] handshaking with SSH
2022/04/06 18:25:25 packer.exe plugin: [DEBUG] SSH handshake err: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none password], no supported methods remain
2022/04/06 18:25:25 packer.exe plugin: [DEBUG] Detected authentication error. Increasing handshake attempts.
2022/04/06 18:25:32 packer.exe plugin: [INFO] Attempting SSH connection to 127.0.0.1:4358...
2022/04/06 18:25:32 packer.exe plugin: [DEBUG] reconnecting to TCP connection for SSH
2022/04/06 18:25:32 packer.exe plugin: [DEBUG] handshaking with SSH
2022/04/06 18:25:34 packer.exe plugin: [DEBUG] SSH handshake err: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none password], no supported methods remain
2022/04/06 18:25:34 packer.exe plugin: [DEBUG] Detected authentication error. Increasing handshake attempts.
2022/04/06 18:25:39 packer.exe plugin: Received interrupt signal (count: 1). Ignoring.
2022/04/06 18:25:39 packer.exe plugin: Received interrupt signal (count: 1). Ignoring.
2022/04/06 18:25:39 packer.exe plugin: Received interrupt signal (count: 1). Ignoring.
2022/04/06 18:25:39 Cancelling builder after context cancellation context canceled
Cancelling build after receiving interrupt
2022/04/06 18:25:39 packer.exe plugin: [WARN] Interrupt detected, quitting waiting for SSH.
2022/04/06 18:25:39 packer.exe plugin: [DEBUG] SSH wait cancelled. Exiting loop.
2022/04/06 18:25:39 packer.exe plugin: Executing VBoxManage: []string{"controlvm", "ubuntu2004-desktop", "poweroff"}
2022/04/06 18:25:40 packer.exe plugin: stdout:
2022/04/06 18:25:40 packer.exe plugin: stderr: 0%...10%...20%...30%...40%...50%...60%...70%...80%...90%...100%
2022/04/06 18:25:40 packer.exe plugin: failed to unlock port lockfile: close tcp 127.0.0.1:4358: use of closed network connection
==> virtualbox-iso: Cleaning up floppy disk...
2022/04/06 18:25:40 packer.exe plugin: failed to unlock port lockfile: close tcp 127.0.0.1:5992: use of closed network connection
2022/04/06 18:25:40 packer.exe plugin: Executing VBoxManage: []string{"storageattach", "ubuntu2004-desktop", "--storagectl", "IDE Controller", "--port", "0", "--device", "1", "--type", "dvddrive", "--medium", "none"}
2022/04/06 18:25:40 packer.exe plugin: stdout:
2022/04/06 18:25:40 packer.exe plugin: stderr:
==> virtualbox-iso: Deregistering and deleting VM...
2022/04/06 18:25:40 packer.exe plugin: Executing VBoxManage: []string{"unregistervm", "ubuntu2004-desktop", "--delete"}
2022/04/06 18:25:40 packer.exe plugin: stdout:
2022/04/06 18:25:40 packer.exe plugin: stderr: VBoxManage.exe: error: Cannot unregister the machine 'ubuntu2004-desktop' while it is locked
2022/04/06 18:25:40 packer.exe plugin: VBoxManage.exe: error: Details: code VBOX_E_INVALID_OBJECT_STATE (0x80bb0007), component MachineWrap, interface IMachine, callee IUnknown
2022/04/06 18:25:40 packer.exe plugin: VBoxManage.exe: error: Context: "Unregister(CleanupMode_DetachAllReturnHardDisksOnly, ComSafeArrayAsOutParam(aMedia))" at line 154 of file VBoxManageMisc.cpp
2022/04/06 18:25:40 packer.exe plugin: Retryable error: VBoxManage error: VBoxManage.exe: error: Cannot unregister the machine 'ubuntu2004-desktop' while it is locked
2022/04/06 18:25:40 packer.exe plugin: VBoxManage.exe: error: Details: code VBOX_E_INVALID_OBJECT_STATE (0x80bb0007), component MachineWrap, interface IMachine, callee IUnknown
2022/04/06 18:25:40 packer.exe plugin: VBoxManage.exe: error: Context: "Unregister(CleanupMode_DetachAllReturnHardDisksOnly, ComSafeArrayAsOutParam(aMedia))" at line 154 of file VBoxManageMisc.cpp
```

I try many thing like making sure the encrypted password here https://github.com/boxcutter/ubuntu/blob/master/http/user-data#L6 is "vagrant"...

I tweak network section as describe here : https://ubuntu.com/server/docs/install/autoinstall-reference 
(2 network keys)

I had ssh PasswordAuthentication yes here : https://github.com/boxcutter/ubuntu/blob/master/script/sshd.sh
(even if I think this happen later in the process)

I change the image reference 20.04.4 is now the up to date version and the old image is not available at the same URL 
20.04.2 : https://old-releases.ubuntu.com/releases/20.04.2/ubuntu-20.04.2-live-server-amd64.iso
20.04.4 : http://www.releases.ubuntu.com/20.04/ubuntu-20.04.4-live-server-amd64.iso

**Comment by BuhtigithuB** [comment]
![image](https://user-images.githubusercontent.com/1660615/162104517-09ae996a-c935-49ac-901c-39598e1189bc.png)

The previous logs were coming from Windows Host, the above screenshot is with Linux Ubuntu 20.04 and VB 6.1.32_Ubuntu r149290

**Comment by BuhtigithuB** [comment]
![image](https://user-images.githubusercontent.com/1660615/162105152-b7809d88-3d3c-4405-82b0-0b6a28f2e3a6.png)

I attempted with windows and linux to ssh in the running installer instance of the to be linux image box, it seems to initiate the ssh communication as you can see in the attached screenshot but when I input the "vagrant" password it fails misirably...

**Comment by BuhtigithuB** [comment]
DEBUG] SSH handshake err: ssh: handshake failed: ssh: unable to authenticate, attempted methods [none password], no supported methods remain

This error is suppose to come from ssh key and password missmatch : 

Here : https://github.com/boxcutter/ubuntu/issues/196#issuecomment-755405452 @nrundle mention that it might be related to `ssh_handshake_attempts` delay that should be longer then the subiquity process to complete or packer will attempt to ssh in the ssh instance of the installer which is not intiated and defined by packer which will trigger the installation to fails... So I tried with longer delay up to 600...

Otherwise the network interface adapter has the right name, I check even it was a concern only for vmware : 

![image](https://user-images.githubusercontent.com/1660615/162106592-fb2308a0-143c-4512-b7ad-167c0648ccc8.png)

**Comment by polobo** [comment]
I was messing around with all this back in August and got the subiquity install completed.  I have a commit where I increased ssh_handshake_attempts from 20 to 200.

**Comment by BuhtigithuB** [comment]
[1649282298.894723177.install_fail.crash.zip](https://github.com/boxcutter/ubuntu/files/8439159/1649282298.894723177.install_fail.crash.zip)


I extracted the crash report joined here...

Hopefully someone with more knowledge than I will make sens of all this :D

**Comment by polobo** [comment]
Probably unrelated but I also added the following to user-data, to prevent hanging.
```
  apt:
    geoip: false
```

**Comment by BuhtigithuB** [comment]
@polobo, I am trying it right now, there is like 3 differents places the handshake attempts can be set... I put all of them to 200...

**Comment by polobo** [comment]
Yeah, I basically started a fork for my limited needs that I could mess with, removing and reworking quite a few things, so a simple share of my changes is less useful (at least without more research on my end).  For reference I got it to work with the following versions:

* Ubuntu 20.04.2 w/ Desktop Install
* Packer: 1.7.4
* Vagrant: 2.2.18
* Oracle VM VirtualBox: Windows 10 - 6.1.26 r145957 (Qt5.6.2)
* Parallels: Not Tested
* VMware Fusion: Not Tested

**Comment by BuhtigithuB** [comment]
It crashed again..

**Comment by BuhtigithuB** [comment]
Just noticed this : 
![image](https://user-images.githubusercontent.com/1660615/162109822-849ebf2f-05de-42eb-8236-c56282dc3ddf.png)

I mean the IP address supply at launch seems bizzare...

**Comment by BuhtigithuB** [comment]
![image](https://user-images.githubusercontent.com/1660615/162109954-813bb9ef-7b0d-4fa2-98f7-31abb55c83eb.png)

The IP address supply at launch seems weird...

**Comment by BuhtigithuB** [comment]
@polobo can you push you hacked branch on your clone... You don't need to make PR just reference your moded branch here with might flag something doing some diff...

I know this whole thing is often version sensitive...

**Comment by polobo** [comment]
https://github.com/polobo/boxcutter-ubuntu/tree/slimdown

**Comment by BuhtigithuB** [comment]
Nice @polobo, you made a lot of improvement using hcl... I will test it, after I try the current repo with packer 1.7.4

**Comment by BuhtigithuB** [comment]
Same old with 1.7.4... Trying your repo now...

**Comment by BuhtigithuB** [comment]
@polobo, you have a small typo not closing double quote in the readme.md 


packer build \
    [-only=<template-type>] \
    [-var-file=ubuntu-add-desktop.pkvars.hcl] \
    -var-file ubuntu{release}.pkvars.hcl \
    -var "version={yyyymmdd-n}**"** \
    build-ubuntu


It is not clear what `version` variable does, my guess is to label created build with different names/versions?

**Comment by polobo** [comment]
It lets me timestamp the artifacts, with an option to build more than one artifact per day by incrementing n {1,2,3,etc}

**Comment by BuhtigithuB** [comment]
That's what I thought...

Saddly it seems to fail on the same issue :

![image](https://user-images.githubusercontent.com/1660615/162223051-72a6e2d7-8558-422f-aedc-810d709a5792.png)

**Comment by BuhtigithuB** [comment]
Is vagrant required to build packer box??

**Comment by polobo** [comment]
> Is vagrant required to build packer box??

I've never tried this particular process without it, and haven't tried any other packer using process either...

My interest is piqued but no guarantees I'm going to be able to dive in deep anytime soon.

**Comment by BuhtigithuB** [comment]
vagrant was installed, so my issue doesn't come from there... I never used vagrant to build boxcutter ubuntu image box...

**Comment by BuhtigithuB** [comment]
https://github.com/hashicorp/packer/issues/8609 

openssh 8.2+ on my system

**Comment by polobo** [comment]
I'm getting the same problem when I try the 20.04-4 ISO.  I unable to experiment further right now but I basically have two thoughts:
Explicitly specify a mirror in user-data:
```
apt:
    geoip: false
    primary:
        - arches: [default]
          uri: "http://mirrors.vcea.wsu.edu/ubuntu"
```

```
--force-yes
           Force yes; this is a dangerous option that will cause apt to continue without
           prompting if it is doing something potentially harmful. It should not be used except
           in very special situations. Using force-yes can potentially destroy your system!
           Configuration Item: APT::Get::force-yes.
```
But figure out how to set that in user-data.
https://curtin.readthedocs.io/en/latest/topics/apt_source.html

Maybe enabling geoip again, assuming the hang doesn't resume, will fix this new issue?

I'm also curious if the 22.04 beta exhibits the same problem.  Toward that end:
```
vm_name = "ubuntu2204"
iso_checksum = "sha256:39fdd5f7e868ab7981b492a8887dbaec85acf798b209162a37893cb4b209a26b"
iso_url = "http://releases.ubuntu.com/22.04/ubuntu-22.04-beta-live-server-amd64.iso"

```

I did re-run the 20.04-2 build with my cached ISO and it worked just fine.

**Comment by polobo** [comment]
I added the primary block to the apt block in the user-data and that didn't change anything.  But removing the entire apt block got me past the subiquity initialization portion and a successful build using:

iso_checksum = "sha256:28ccdb56450e643bad03bb7bcf7507ce3d8d90e8bf09e38f6bd9ac298a98eaad"
iso_url = "http://releases.ubuntu.com/20.04/ubuntu-20.04-4-live-server-amd64.iso"

Same software architecture I showed earlier, though the error seems to exist within the iso itself so I'm doubtful software versions for the other components matter.
